requirements.in

```diff
@@ -7,3 +7,5 @@
 pygls
 packaging
 pylint
+flake8
+flake8-pyproject
```

Comment:
Only packages that we ship in the release package should go in here. we can create a `dev_requirements.in` that includes the `wheel`, `nox`, `pip-tools` and these.

Reply:
fixed.